packages/connect/tsconfig.test.json

```diff
@@ -1,14 +1,16 @@
 {
     "compileOnSave": true,
     "compilerOptions": {
-        "target": "ES2018",
+        "target": "ESNEXT",
```

Comment:
why next and not 2020?

Reply:
For now - ESNEXT and ES2020 are same - but to be on safe side I will change that to ES2020